versioned_docs/version-3.10/use-cases/custom-pipelines-flow.md

```diff
@@ -0,0 +1,376 @@
+    │   │   ├── bitbucket-build-lib-default.yaml
+    │   │   ├── bitbucket-build-lib-edp.yaml
+    │   │   ├── bitbucket-review.yaml
+    │   │   ├── gerrit-build-default.yaml
+    │   │   ├── gerrit-build-edp.yaml
+    │   │   ├── gerrit-build-lib-default.yaml
+    │   │   ├── gerrit-build-lib-edp.yaml
+    │   │   ├── gerrit-review.yaml
+    │   │   ├── github-build-default.yaml
+    │   │   ├── github-build-edp.yaml
+    │   │   ├── github-build-lib-default.yaml
+    │   │   ├── github-build-lib-edp.yaml
+    │   │   ├── github-review.yaml
+    │   │   ├── gitlab-build-default.yaml
+    │   │   ├── gitlab-build-edp.yaml
+    │   │   ├── gitlab-build-lib-default.yaml
+    │   │   ├── gitlab-build-lib-edp.yaml
+    │   │   └── gitlab-review.yaml
+    │   ├── resources
+    │   │   └── npm-settings.yaml
+    │   ├── tasks
+    │   │   ├── deploy
+    │   │   │   └── hello-world-deploy.yaml
+    │   │   └── hello-world.yaml
+    │   └── triggers
+    │       ├── custom-clean.yaml
+    │       └── custom-deploy.yaml
+    └── values.yaml
+    ```
+
+    The `templates` directory is a main directory that contains templates for custom Tekton pipelines, tasks, and trigger templates. This directory is divided into subdirectories for pipelines, tasks, and triggers.
+
+    - The `templates/pipelines` directory contains Tekton pipeline templates for build and review processes. For each Git server (e.g., GitHub, GitLab, Bitbucket, or Gerrit), there are specific templates, organized by component and versioning type.
+      - The appropriate template should be selected based on the configuration of your component. For example:
+        - If the component is created in GitHub, has the `Application` component type, and uses `edp` versioning, you should use the `github-build-edp.yaml` template.
+        - If the component is created in Bitbucket, has the `Library` component type, and uses the `default` versioning, you should use the `bitbucket-build-lib-default.yaml` template.
+
+        Here is an example illustrating the naming conventions used in template filenames:
+
+        ![Pipeline Templates](../assets/use-cases/custom-tekton-pipelines/file-structure.png "File name structure")
+
+    - The `templates/pipelines/deploy` directory contains Tekton pipeline templates for deploy and clean processes. Select the appropriate template based on the deployment process requirements.
+    - The `templates/tasks` directory contains Tekton task templates for specific tasks that are part of the build, review, or deployment processes.
+    - The `templates/triggers` directory contains Tekton trigger templates for triggering the deployment or clean processes.
+    - The `templates/resources` directory is intended for various templates, such as config maps and secrets, which can be mounted into custom Tekton tasks as needed.
+
+5. Modify the existing pipeline and task templates or create new ones based on your project requirements.
```

Comment:
here we need to higlight what we have as a result of this part of case and how we can use